### ISSUE TITLE
Do disco#info query and fetch VCard when user subscribes to a new JID

### ISSUE THREAD

**Issue by jcbrand** [conversejs/converse.js#1361]
When a user enters the JID of someone they'd like to add as a contact, Converse can do a disco#info query to check whether the account exists and can additionally fetch metadata such as the VCard to present to the user.

**Comment by linkmauve** [comment]
It’s easy to obtain public information about a user JID, but not so much to be certain it doesn’t exist.

**Comment by dwd** [comment]
True, the server could just provide dummy info. But in general, vCards are fairly public things, and do help a lot.
